### PR TITLE
Update from auto-update/renku-core-2.3.2

### PR DESCRIPTION
**Pull request by RenkuBot**
This is an automated pull request.

/deploy #persist

### CODE REVIEW

**Review by aledegano — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,36 @@
 .. _changelog:
 
+0.24.2
+------
+
+Renku ``0.24.2`` introduces bug fixes.
```
Comment (lokijuhy) [inline CHANGELOG.rst:6]:
```suggestion
Renku ``0.24.2`` introduces bug fixes. We squashed a bug where you may have had trouble finding your projects in other namespaces. Now, all projects show up in the Knowledge Graph activation page so you can add them to your dashboard.
```

Is this an accurate summary?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+~~~~~~~~~~~~~~~~~~~
+
+**Improvements**
+* **Knowledge Graph**: A new Delete Project API
+
+**🐞 Bug Fixes**
+
+* **Core Service**: Fix a crash when migrating to v10 metadata
+  (`#3359 <https://github.com/SwissDataScienceCenter/renku-python/pull/3359>`__).
+* **Knowledge Graph**: The Inactive Projects page to list all projects where logged user is a member
```
Comment (lokijuhy) [inline CHANGELOG.rst:18]:
```suggestion
* **Knowledge Graph**: Expand the Knowledge Graph Project Activation page to list all projects where the user is a member, not only owned projects.
```

**Comment by rokroskar** [comment]
How did this get merged without @SwissDataScienceCenter/renku-product-management approval??

**Comment by lokijuhy** [comment]
> How did this get merged without @SwissDataScienceCenter/renku-product-management approval??

Yea, I only just realized I commented on an already-merged PR... ??

**Comment by lokijuhy** [comment]
> How did this get merged without @SwissDataScienceCenter/renku-product-management approval??

@rokroskar Oh there's a conversation in Slack...

**Pull request merged**